rholang/src/main/scala/coop/rchain/rholang/interpreter/matcher/package.scala

```diff
@@ -17,7 +17,8 @@ package object matcher {
   type OptionalFreeMapWithCost[A] = StateT[OptionWithCost, FreeMap, A]
   type OptionWithCost[A]          = OptionT[ErroredOrCostA, A]
 
-  type ErroredOrCostA[A] = StateT[Either[OutOfPhlogistonsError.type, ?], Cost, A]
+  type OOPE[A] = Either[OutOfPhlogistonsError.type, A]
```

Comment:
What does `OOPE` stand for?

Reply:
OutOfPhlogistonError. If you want a rename, propose sth :)

Reply:
So what's the difference between `OOPE` a.k.a `OutOfPhlogistonError` and `OutOfPhlogistonsError`? :-)

Reply:
I think you can very well read that difference from line 20 (-:

Reply:
My point is that there is a big semantic difference between those types, but their names merely differ by `s`

Reply:
Disregard the `s` - I meant `OutOfPhlogistonsError`

Reply:
This isn't converging, moved to discord